src/bin/utils/parsers.ts

```diff
@@ -21,6 +21,25 @@ function validateParserToArgParser<T>(
   };
 }
 
+/**
+ * Converts a validation parser to commander variadic argument parser
+ */
```

Comment:
Can you make a comment about how variadic options/arguments are always space separated.

I would also just prototype and see what happens if you were to do something like `--variadic-option 'a b c' '123'` and `--variadic-option='a b c' --variadic-option=123`, to see how it handles spaces that are within the same item.

Reply:
Added a comment - will add tests for this when I continue test splitting in #311.